### ISSUE TITLE
Check UDP errors after increasing IP ban duration from 2 minutes to 1 hour

### ISSUE THREAD

**Issue by josecelano** [torrust/torrust-demo#28]
Relates to: 
  - https://github.com/torrust/torrust-tracker/issues/1139
  - https://github.com/torrust/torrust-tracker/pull/1124#issuecomment-2550833270

IP bans were increased from 120 seconds to 1 hour.

IPs are banned when they send more than 10 requests with the wrong connection ID.

You can see grafana dashboard before this change [here](https://github.com/torrust/torrust-tracker/pull/1124#issuecomment-2550833270).

![image](https://github.com/user-attachments/assets/b0eca1c0-909d-4bb2-81c3-6c2306399369)

The change was merged [here](https://github.com/torrust/torrust-tracker/pull/1144) at 2024-12-19 12:26 UTC.

We were banning 100 req/sec on average. I will run the tracker with the new configuration for hours to compare it with the previous data.

**Comment by josecelano** [comment]
After 12 hours of running the 1-hour ban:

![image](https://github.com/user-attachments/assets/f1d01eb2-95aa-4b62-a599-928b11c050f9)

There are fewer UDP errors as I expected. However, I don't know why the connection requests graph seems to follow error graphs. Why are connections decreasing when we ban IPs because they send wrong connection IDs? That means clients make a connection request but do not send the right one in the following announce/scrape requests. But error since to be equally distributed between:

- cookie value is expired
- cookie value is from future

If the problem is that clients are reusing the same connection ID for too long, we will only have `cookie value is expired` errors.

Maybe after running this experiment https://github.com/torrust/torrust-demo/issues/29 we can know more.

[I have also analysed the behaviour of one banned client](https://github.com/torrust/torrust-demo/issues/14#issuecomment-2488337204). That client was making the `connect` request but then it was always using the same connection ID:

```text
ConnectionId(I64(-2325979768337461225))
```

But I don't remember if there was an announcement request returning that ID. I don't think so or it was made before the date range I got from logs.
